### ISSUE TITLE
Frames will not load

### ISSUE THREAD

**Issue by Specom** [AutomaticBehaviorRecognitionSystem/ABRS#3]
Hey there,

I am currently trying to label videos in a fresh annaconda enviroment in windows using the latest GUI, however, the videos can not be loaded. I tried mp4 and avi format and different videos and locations. Are there some codecs or so missing or do I need other installations in annaconda? 

Cheers

Sander

**Comment by Specom** [comment]
Just a short update: running the video with the following code in jupiter lab works:
from ipywebrtc import VideoStream
video = VideoStream.from_file('C:/Users/X/ABRS-master/TEST_PROJECT/2024-05-14_A03_009_camera_01.mp4')
video

In ABRS there is no progress in loading some frames for a long time and no error code is provided

**Comment by Specom** [comment]
2024-05-17 14:36:53.624986: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-05-17 14:36:54.151151: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1616] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 9597 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 3060, pci bus id: 0000:09:00.0, compute capability: 8.6
